content/applications/hr/payroll/configuration.rst

```diff
@@ -0,0 +1,356 @@
+=====================
+Payroll configuration
+=====================
+
+Odoo *Payroll* is used to process work entries and create payslips for employees, including both
+regular pay and commission. Payroll works in conjunction with other Odoo apps, such as *Employees*,
+*Timesheets*, *Time Off*, and *Attendances*.
+
+The *Payroll* app helps ensure there are no issues or conflicts when validating work entries,
+handles country-specific localizations to ensure that payslips follow local rules and taxes, and
+allows for salary assignments. Payroll configuration is critical to ensure accurate and timely
+processing of payslips.
+
+Settings
+========
+
+To access the settings, go to :menuselection:`Payroll --> Configuration --> Settings`. Whether or
+not payslips are posted in accounting, and whether SEPA payments are created, is selected here.
+
+The :guilabel:`Settings` section is where localization settings are configured. *Localizations* are
+country-specific settings pre-configured in Odoo at the creation of the database, and account for
+all taxes, fees, and allowances for that particular country. The localization section of the
+settings includes a detailed view of all benefits provided to employees.
+
+.. image:: configuration/payroll-settings.png
+   :align: center
+   :alt: Settings available for Payroll.
+
+Any country-specific localizations are set up in the :guilabel:`Localization` section of the
+settings screen. All localization items are pre-populated when the country is specified during the
+creation of the database. It is not recommended to alter the localization settings unless
+specifically required.
+
+Work entries
+============
+
+A *work entry* is an individual record on an employee's timesheet. Work entries can be configured to
+account for all types of work and time off, such as :guilabel:`Attendance`, :guilabel:`Sick Time
+Off`, :guilabel:`Training`, or :guilabel:`Public Holiday`.
+
+Work entry types
+----------------
+
+When creating a work entry in the *Payroll* application, or when an employee enters information in
+the *Timesheets* application, a :guilabel:`Work Entry Type` needs to be selected. The list of
+:guilabel:`Work Entry Types` is automatically created based on localization settings set in the
+database.
+
+To view the current work entry types available.Go to :menuselection:`Payroll --> Configuration -->
+Work Entry Types`
+
+Each work entry type has a code to aid in the creation of payslips, and ensure all taxes and fees
+are correctly entered.
```

Comment:
Yeah these are tough, since I feel the need to explain the why, then the how, and I am using the terms in the UI.  I also wonder what @StraubCreative says on this.

Reply:
Yeah these are tough, since I feel the need to explain the why, then the how, and I am using the terms in the UI.  I also wonder what @StraubCreative says on this.